datastore/exporters/nmdb-export-port-list/nmdb-export-port-list.cpp

```diff
@@ -118,6 +118,13 @@ class Tool : public nmdt::AbstractExportTool
             NULL_SEMANTIC,
             "Use ports from database instead of from resource file.")
           );
+          
+      opts.addOptionalOption("config-path", std::make_tuple(
+            "config-path,c",
+            po::value<std::string>(),
+            "Use specified port configuration file instead of the default.")
+          );
```

Comment:
Instead of hiding what the default path is, we will want to place it as the default when they display the help messages.  For example,
```suggestion
      auto& nmfm {nmcu::FileManager::getInstance()};
      opts.addOptionalOption("config-path", std::make_tuple(
            "config-path,c",
            po::value<std::string>()->default_value(nmfm.getConfPath()/"port-list.conf"),
            "Use specified port configuration file instead of the default.")
          );
```

This also means the changes below need to be fixed/undone/changed since that logic isn't needed or is invalid now.